Gruntfile.js

```diff
@@ -25,7 +25,6 @@ module.exports = function( grunt ) {
 	// ⚠️ Warning: These paths are passed straight to rm command in the shell, without any escaping.
 	const productionVendorExcludedFilePatterns = [
 		'composer.*',
-		'lib',
 		'vendor/*/*/.editorconfig',
 		'vendor/*/*/.git',
```

Comment:
The `.github` folder in `ampproject/amp-toolbox` can be excluded. At the same time since it's a common folder let's ignore it for all dependencies.

```suggestion
		'vendor/*/*/.git',
		'vendor/*/*/.github',
```